packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-type-form.tsx

```diff
@@ -1,60 +1,365 @@
+            },
+            validate(value) {
+              setDescriptionValid(!!value);
+
+              return value ? true : "You must choose a description";
+            },
+          })}
+        />
+        <Controller
+          render={({ field: { onChange, ...props } }) => (
+            <Autocomplete
+              multiple
+              popupIcon={null}
+              clearIcon={null}
+              forcePopupIcon={false}
+              selectOnFocus={false}
+              openOnFocus
+              clearOnBlur={false}
+              onChange={(_evt, data) => onChange(data)}
+              {...props}
+              renderTags={(value, getTagProps) =>
+                value.map((option, index) => (
+                  <Chip
+                    {...getTagProps({ index })}
+                    key={option.dataTypeId}
+                    label={
+                      <Typography
+                        variant="smallTextLabels"
+                        sx={{ display: "flex", alignItems: "center" }}
+                      >
+                        <FontAwesomeIcon
+                          icon={{ icon: option.icon }}
+                          sx={{ fontSize: "1em", mr: "1ch" }}
+                        />
+                        {option.title}
+                      </Typography>
+                    }
+                    color="blue"
+                  />
+                ))
+              }
+              renderInput={(inputProps) => (
+                <TextField
+                  {...inputProps}
+                  label="Expected values"
+                  sx={{ alignSelf: "flex-start", width: "70%" }}
+                  placeholder="Select acceptable values"
+                />
+              )}
+              options={propertyTypeDataTypes}
+              getOptionLabel={(obj) => obj.title}
+              disableCloseOnSelect
+              renderOption={(optProps, opt) => (
+                <Box component="li" {...optProps} sx={{ py: 1.5, px: 2.25 }}>
+                  <FontAwesomeIcon
+                    icon={{ icon: opt.icon }}
+                    sx={(theme) => ({ color: theme.palette.gray[50] })}
+                  />
+                  <Typography
+                    variant="smallTextLabels"
+                    component="span"
+                    ml={1.5}
+                    color={(theme) => theme.palette.gray[80]}
+                  >
+                    {opt.title}
+                  </Typography>
+                  <Chip color="blue" label="DATA TYPE" sx={{ ml: 1.5 }} />
+                </Box>
+              )}
+            />
+          )}
+          control={control}
+          rules={{ required: true }}
+          name="expectedValues"
+        />
+      </Stack>
+      <Divider sx={{ mt: 2, mb: 3 }} />
+      <Stack direction="row" spacing={1.25}>
+        <Button
+          {...createButtonProps}
+          loading={isSubmitting}
+          disabled={isSubmitting || !!nameError}
```

Comment:
I suggest using `disabled={isSubmitting || !isValid}` here. If we're showing validation errors as expected, we can keep the button disabled I guess. 

Here for example, when we try to submit with an empty `expectedValues` array, `handleSubmit` will not be invoked (since there is an validation error at `expectedValues`, but UI does not represent this correctly. I think keeping the button disabled if form is not valid could be a good way of representing this.)

Reply:
[`c3e17d6` (#1345)](https://github.com/hashintel/hash/pull/1345/commits/c3e17d6591f1629322dce7ac9defd03860d7fdb0)